src/olympia/scanners/admin.py

```diff
@@ -543,6 +543,11 @@ class ScannerQueryRuleAdmin(AbstractScannerRuleAdminMixin, admin.ModelAdmin):
         'created', 'modified', 'matched_results_link', 'state_with_actions',
     )
 
+    def has_change_permission(self, request, obj=None):
+        if obj and obj.state != NEW:
```

Comment:
I know it's not introduced in this patch, but imo these state constants are too generically named and should be either be prefixed e.g. `SCANNER_STATE_NEW` or at least namespaced. e.g. `from olympia.constants import scanners`

Reply:
filed https://github.com/mozilla/addons-server/issues/13411